andela/travel_tool_front
# #160127329 Refactor Dockerfile and Makefile

#### What does this PR do?
- Removes line that copies entire project to docker image used for development
- Builds docker image before running tests

#### Description of Task to be completed?
1. Dockerfile, used for development purposes, is copying entire root folder into the container while it only needs to mount it as a volume. Remove the line that copies entire root folder into the container.

2. Currently, when running tests inside the container using `make test` command, it is NOT building the image, and as a result, if there are any changes to `package.json` file, they would go unnoticed and tests may fail. Update the makefile so that when `make test` command is ran, it also builds the docker image.

#### How should this be manually tested?
Run `make test` command on the terminal and it should first build the docker image.

#### What are the relevant pivotal tracker stories?
#160127329
